packages/teleport/src/lib/tdp/sharedDirectoryManager.ts

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+import { FileType } from './codec';
```

Comment:
Plz be explicit when importing types.

```suggestion
import type { FileType } from './codec';
```

Reply:
It's used as an object here

https://github.com/gravitational/webapps/blob/189c1d4ed84ed7a282c7028239ac3ce199f4e668/packages/teleport/src/lib/tdp/sharedDirectoryManager.ts#L166

Reply:
Ahh I see now, thanks sorry I missed that.